pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "tox-poetry-installer"
-version = "0.8.4"
+version = "1.0a1"
```

Comment:
I'd rather we bump to `0.9` here rather than 1.0. [The roadmap](https://github.com/enpaul/tox-poetry-installer#path-to-stable) still highlights dropping the `poetry` dependency entirely as a requirement for the 1.0 release and I'd like to keep that the case. Having the plugin require poetry but only as an optional dependency is kind of a gross work around that we need for now until the required functionality gets added to poetry-core. Ideally, once we support poetry 1.2, that can be the next big step.